rust/build.rs

```diff
@@ -1,16 +1,41 @@
 extern crate bindgen;
+extern crate pkg_config;
 
-use std::env;
 use std::path::PathBuf;
+use std::process::Command;
 
 fn main() {
-    // Tell cargo to tell rustc to link the system bzip2
-    // shared library.
-    println!("cargo:rustc-link-search=native={}", "../lib");
-    println!("cargo:rustc-link-lib=ceed");
+    let out_dir = PathBuf::from(env("OUT_DIR").unwrap());
+    let statik = env("CARGO_FEATURE_STATIC").is_some();
+
+    let mut pkgconf = pkg_config::Config::new();
+    if env("CARGO_FEATURE_SYSTEM").is_none() {
```

Comment:
I think somewhere in here the GitHub action is reading an extra newline character

Reply:
I don't know what exactly you were commenting on, but the action is clean now and I think cleaner.